README.md

```diff
@@ -2,11 +2,69 @@
 
 Este proxecto é unha web de referencia para o grupo _Hello Sisters!_.
 
-O modelo que se empregou foi [este](https://html5up.net/dimension)
+O modelo que se empregou foi [GameR - gaming HTML template](https://themehunt.com/item/1526056-gamer-gaming-html-template). 
 
-Se queres facer calquera modificación, asegúrate de que as imaxes que poñas sexan Creative Commons ou de citares a súa autoría se a/o propietaria/o así o indica.
+Se queres facer calquera modificación, asegúrate de que as imaxes que poñas sexan Creative Commons ou de citares a súa autoría se a/o propietaria/o así o indica. Polo de agora, algunhas das imaxes que se usan de fondo na web foron sacadas de [Unsplash](https://unsplash.com/).
 
 Por outra parte, para calqueira modificación, fai unha *Pull Request* ao repositorio, creando a túa propia _branch_.
 
-Non fagas cambios directos no _master_! ;)
+Non fagas cambios directos na rama _main_! ;)
 
+_______________________
+
+## Como colaborar neste repositorio?
+
+### Metodoloxía de traballo:
+
+Os pasos que seguimos cando queremos modificar algo do repositorio son os seguintes:
+
+1. Crear tarefa/ticket no backlog do [proxecto en GitHub](https://github.com/hellosisters/hellosisters.github.io/projects/1) coa descrición da tarefa que se vaia facer.
+2. Mover a tarefa/ticket a _In progress_, crear un _issue_ a partir del e asignalo a quen corresponda (posiblemente sexas ti quen vai realizar esa nova tarefa). 
+
+   + Se o consideras necesario, pódeslle poñer a etiqueta que mellor che pareza.
+   
+   + Dentro dese _issue_ é moi práctico e recomendable engadir a descrición da tarefa, onde tamén podes engadir _checkboxes_, de forma que se poida ir marcando o progreso.
+3. Crear la rama en local ( feature/_nomeDaRama_ ).
+4. Unha vez remates todo o que tiñas pensado facer para esa tarefa, so quedaría subila e crear unha PR (Pull Request) - [guía para crear unha PR](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
```

Comment:
Modifiquei as columnas. Tiñamos "to do" e "backlog" que un pouco é o mesmo para este proxecto. Deixei "backlog" e creei "in test", polo que cando se remate de traballar nun ticket e a PR esté aberta, deberíase mover o ticket a "in test".